Change item/xp merge and entity tracking range defaults

kennytv opened the pull request:
Restore vanilla defaults for the merge radius, increase the tracking range values as the current ones are absolute ass. I also increased `raiderActivationRange`, as raiders like to get stuck just out of reach and become unfindable.

Waiting on feedback on possibly changing other config values. No Paper comments because it's just a Spigot file and Paper go zoom

kennytv committed:
Change item/xp merge and entity tracking range defaults

masmc05 commented on patches/server/0005-Paper-config-files.patch:
```diff
@@ -5111,6 +5111,47 @@ index a6d2ce801b236b046b94913bccf7eccfc561f35a..b244b65799d9c082b8b1639bad157274
++    public int playerTrackingRange = 128;
++    public int animalTrackingRange = 96;
++    public int monsterTrackingRange = 96;
++    public int miscTrackingRange = 96;
```
Why can't tracking ranges be -1 by default, which will result in using vanilla values?

Vanilla has various values for each mob, setting a universal value per category already will result in behaviour differences without any possibility to restore it

kennytv replied:
The purpose of this pr is to improve defaults, not to necessarily make them equal to vanilla, and those are visual only anyways

Machine-Maker commented on patches/server/0005-Paper-config-files.patch:
```diff
@@ -5111,6 +5111,47 @@ index a6d2ce801b236b046b94913bccf7eccfc561f35a..b244b65799d9c082b8b1639bad157274
+     private void expMerge()
+     {
+-        this.expMerge = this.getDouble("merge-radius.exp", 3.0 );
++        this.expMerge = this.getDouble("merge-radius.exp", 0.5 );
```
It was pointed out that the vanilla value for this is actually to disable it completely with `-1`. There are 2 exp merge systems in the server, this only controls the custom one spigot added, and doesn't mess with vanilla's own merge system. So disabling this completely will result in vanilla xp behavior.

EterNityCH replied:
Oh I was not aware of this and yeah that would certainly be a more favorable result.

kennytv committed:
Disable Spigot xp merging by default
Players need to deal the final blow for mobs to drop xp orbs and will generally be picked up rather quickly. For larger masses, vanilla has its own merging
